app/src/main/res/values/strings.xml

```diff
@@ -6,6 +6,7 @@
     <string name="my_story">Mano istorija</string>
     <string name="no_more_pets_to_swipe">O ne!\nSąrašas jau baigėsi.\nPatikrink pamėgtų gyvūnų sąrašą</string>
     <string name="empty_favorites_list">Neturite pamėgtų gyvūnų,\nlaikas išssirinkti draugą!</string>
+    <string name="spausk_susisiek">Spausk ir susisiek\nsu prieglaudos atstovu!</string>
```

Comment:
Let's use english variable names.

Reply:
What would you think about adding more information about mentorship ? E.g `Spausk ir susisiek\nsu prieglaudos atstovu!\nNepamiršk paminėti apie GetPet ir gauk mentorių.`

Reply:
Good suggestions, followed both.